templates/eap-xp1-third-party-db-s2i.json

```diff
@@ -0,0 +1,721 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,eap-xp,javaee,java,jboss",
+            "version": "1.0",
+            "openshift.io/display-name": "JBoss EAP XP 1.0 DB driver examples",
+            "openshift.io/provider-display-name": "Red Hat, Inc.",
+            "description": "An example JBoss Enterprise Application Platform application. For more information about using this template, see https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/7.3.x/README.adoc.",
```

Comment:
Example of a JBoss EAP XP application with a 3rd party database. Fore more information...

Reply:
URL must be https://github.com/jboss-container-images/jboss-eap-openshift-templates/blob/eap-xp1/README.adoc